spec/classes/node_exporter_spec.rb

```diff
@@ -34,7 +34,7 @@
         end
       end
 
-      context 'without collector parameters' do
```

Comment:
Why did this change?

Reply:
ah, as I said, I copied the node_exporter class etc. and realised, that the name of this context is wrong.
It should say `with` as far as I see because it tests _with_ collectors_enabled and collectors_disabled option. Same in line 58.

I guess this was just a copy paste error of the created on the node_exporter spec. Probably copied from `context 'without parameters' do`

Reply:
@ghoneycutt is your questions sufficiantly answered?

Reply:
This spec file is not related to this PR so it should be in a separate PR. This makes it easy for people to understand and merge.

Reply:
done